db/fixtures/answers.yml

```
@@ -27,3 +27,10 @@ answer6:
   description: Q&Aのコメントです。(ベストアンサーではない)
   user: machida
   question: question6
+
+  <% 7.upto(50) do |i| %>
+answer<%= i %>:
+  description: ページャー確認用のデモ回答です。
+  user: komagata
+  question: question1
+  <% end %>
```

```suggestion
<% end %>
```

doとendの間の文字が繰り返されるのでこっちの方がいいかもです〜